### DISCUSSION TITLE
chia wallet get_transactions does not get all transactions?

### DISCUSSION THREAD

**Discussion by jayhohoho2019** [Chia-Network/chia-blockchain#8071] [Q&A]
Hello, what are the expected results of "chia wallet get_transactions".  All historical transactions, a certain number of transactions? I understand it pages through the results.  It seems older transactions are not returned.  And the "Created at" timestamp for a transaction could actually change?

**Comment by keliew** [comment]
`chia wallet get_transactions` will `Get all transactions`. You just need to `Press...c to continue` to see older transactions.

`Created at` cannot be changed.

**Comment by BrandtH22** [comment]
@jayhohoho2019 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
